tests/test_commands.py

```
@@ -83,30 +83,30 @@ async def test_upload_command_valid_csv(
-        mock_interaction, contest_id=1, attachment=attachment
-    )
-
-    # Assert
-    mock_interaction.followup.send.assert_called_once()
-    args, _ = mock_interaction.followup.send.call_args
-    assert expected_error_part in args[0]
-    mock_crud.bulk_create_matches.assert_not_called()
+    # Use context managers for patching to reduce function args
+    with patch("src.commands.matches.crud") as mock_crud, patch(
+        "src.commands.matches.get_session"
+    ) as mock_get_session:
```

Ruff (0.14.10)

Local variable `mock_get_session` is assigned to but never used

Remove assignment to unused variable `mock_get_session`

(F841)